src/core/mantle/types/appearance.ts

```diff
@@ -139,6 +140,16 @@ export type ResourceAppearance = {
   clampToGround?: boolean;
 };
 
+export type RasterAppearance = {
+  url?: string;
```

Comment:
```suggestion
```

url should be handled on `data`. URL is not style or appearance of raster. (glTF has no geo info and it cannot be handled as a feature, but wms can produce features, so it's data)